helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -93,24 +93,6 @@ rules:
       - create
       - delete
       - update
-  - apiGroups:
```

Comment:
By removing the legacy controller only used on KVM, we are not using AwsConfig and AzureConfigs